src/types/Reader.ts

```diff
@@ -110,11 +110,11 @@ export namespace Reader {
   export type ConnectionStatus = 'connected' | 'connecting' | 'notConnected';
 
   export type DisconnectReason =
-    | 'disconnectRequested'
-    | 'rebootRequested'
-    | 'securityReboot'
-    | 'criticallyLowBattery'
-    | 'poweredOff'
-    | 'bluetoothDisabled'
+    | 'disconnect requested'
+    | 'reboot requested'
+    | 'security reboot'
+    | 'critically low battery'
+    | 'powered off'
+    | 'bluetooth disabled'
```

Comment:
we usually use camelCase for these strings, can we keep these as `disconnectRequested` etc and change the mappers in the native layer to follow that format? @ianlin-bbpos

Reply:
ok, I'll do it.